### ISSUE TITLE
some more examples would be welcomed ; ) 

### ISSUE THREAD

**Issue by christophe-g** [crossfilter/universe#39]
Hi,
I really appreciate the effort of lowering the barrier of entry to crossfilter - in particular providing some kind of grammar for querying multivariate dataset. However, playing around the library is not extremely straightforward at the moment, and I found that its added value over defining your own dimensions and groups in crossfilter directly was not obvious from the provided code samples. 

Maybe porting the [Airline on-time performance](http://crossfilter.github.io/crossfilter/) example to universe could help a user like myself see/understand/dwelve into a more comprehensive usecase (e.g. filtering the dataset on multiple dimensions, see how we can add listeners to filtering actions, ...  ). 

I actually tried to port this example myself - without success (at least given the timeframe I allowed myself to test out the lib - and the available amount of brain cells...).

Thanks anyway,
C.

P.S. There are some helpful hints/example in the gitter channel though - saw them a bit too late!
P.P.S. Oh - this should be highlighted ! http://hypercubed.github.io/Project-Chi/#/examples/universe

**Comment by christophe-g** [comment]
http://codepen.io/christophe-g/pen/ALqvQZ

It is possible to do better - but should still be useful if someone gets the same first impression than me,
Cheers,
C.

**Comment by RedShift1** [comment]
Would definitely like more examples too... ELI5/handholding...
